packages/core/src/config/load.ts

```diff
@@ -62,11 +62,17 @@ async function addConfigMetadata({
 }
 
 export async function loadConfig(
-  configPath: string | undefined = findConfig(),
-  customExtends?: string[],
-  processRawConfig?: (rawConfig: RawConfig) => void | Promise<void>
+  options: {
+    configPath?: string;
+    customExtends?: string[];
+    processRawConfig?: (rawConfig: RawConfig) => void | Promise<void>;
+    files?: string[];
```

Comment:
Do we need it here? I mean, the function name is `loadConfig` and seems to add external config values. How about making a new function for that? 

I don't know if this can be done easily. This is just an idea. Non-blocking.